sklearn/linear_model/tests/test_sgd.py

```diff
@@ -31,6 +31,11 @@
 from sklearn.model_selection import RandomizedSearchCV
 
 
+# 0.23. warning about tol not having its correct default value.
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:max_iter and tol parameters have been")
```

Comment:
There was so many warnings in this file that I globally ignored them, for the whole module.